docs: add "ivy" to known scopes in contributing guide [patch]

devversion opened the pull request:
Currently the contributing guide misses the `ivy` scope. This
commit adds `ivy` to the contributing guide as it is useful
for new contributors to know which scopes are supported.

**Note**: this is the patch version of #31291

gkalpak commented on CONTRIBUTING.md:
```diff
@@ -213,6 +213,7 @@ The following is the list of supported scopes:
 * **elements**
 * **forms**
 * **http**
+* **ivy**
```
This is not a package namec (see the description above). It should go in the "exceptions" list below.

devversion replied:
Done.

gkalpak reviewed (changes requested):
:100:

(BTW, when targeting `patch-only`, it is better to select the patch branch as target for the PR :wink:)

devversion committed:
docs: add "ivy" to known scopes in contributing guide

Currently the contributing guide misses the ivy scope. This
commit adds ivy to the contributing guide as it is useful
for new contributors to know which scopes are supported.

Note: this is the patch version of #31291

devversion commented:
@gkalpak Addressed your comment. Also rebased on top `8.0.x` as the commit before was based on `master`. Unfortunately it temporarily triggered all code owners since the PR was set to `master` 😞

devversion commented:
I'm not sure why the Googlebot complains now even though the only commit is from my account (with proper commit e-mail)... closing as this PR is just in a bad state now..
